foundation/foundation-maven-plugins/mda-maven-plugin/src/main/java/com/boozallen/mda/maven/mojo/PipelineArtifactsMojo.java

```diff
@@ -12,6 +12,7 @@
 
 import com.boozallen.aiops.mda.generator.common.PipelineImplementationEnum;
 import com.boozallen.aiops.mda.generator.util.SemanticDataUtil;
+import com.boozallen.aiops.mda.metamodel.AIOpsModelInstanceRepostory;
 import com.boozallen.mda.maven.ArtifactType;
```

Comment:
I'm assuming `aiops` is to remain for the packages in lines 13 - 15.

Reply:
Yes, this a Java module. This PR is to focus on Python modules.